superset-frontend/packages/superset-ui-core/test/chart/components/ChartDataProvider.test.tsx

```diff
@@ -16,16 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
-import { ReactNode } from 'react';
-import { shallow } from 'enzyme';
+import '@testing-library/jest-dom';
+import { render, screen, act } from '@testing-library/react';
```

Comment:
Comment to this, and probably every other file - import from `spec/helpers/testing-library` and you won't need to import additional stuff like `'@testing-library/jest-dom'` 🙂

Reply:
Done! I even added a linting rule to check for these :D